models/demos/whisper/tests/test_performance.py

```diff
@@ -95,7 +87,10 @@ def test_performance(device, use_program_cache, model_name, batch_size, sequence
         comments="",
         inference_time_cpu=0.0,
     )
-
     logger.info(f"Compile time: {inference_and_compile_time - inference_time}")
     logger.info(f"Inference time: {inference_time}")
     logger.info(f"Samples per second: {1 / inference_time * batch_size}")
+
+    assert (
+        inference_time < expected_inference_time
+    ), f"Expected inference time: {expected_inference_time} Actual inference time: {inference_time}"
```

Comment:
If this test is run in the model perf pipeline, it should already check for this via the perf report prepared above after all tests run, making this assert unnecessary

Reply:
Got it. 
Removed assertion.